### PR TITLE
Update decidim version to 0.26

### PR DESCRIPTION
**Pull request by ClaudiaMoPr**
This PR upgrades the application to Decidim version 0.26.
Release notes: https://github.com/decidim/decidim/releases/tag/v0.26.0

On deploy it will require to execute the following commands:

- Upgrade server's Ruby version to 2.7.5
- For locked users are present in global search, to update the search and make them disappear, Run in a bin/rails console or create a migration with:
  `Decidim::User.find_each(&:try_update_index_for_search_resource)`
- For fixing statistic in comments. There's a rake task that you need to run:
  `bin/rails decidim_comments:update_participatory_process_in_comments`
- For Base64 images migration run the following task:
  `bin/rails decidim:active_storage_migrations:migrate_inline_images_to_active_storage[admin_email]`

### CODE REVIEW

**Review by laurajaime — APPROVED** [review]

**Review by laurajaime — CHANGES_REQUESTED** [review]
I do it QA but there is a missing translation, can you check it?

**Comment by laurajaime** [comment]
@ClaudiaMoPr there is the error:

image

**Review by laurajaime — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -4,26 +4,21 @@ source "https://rubygems.org"
...
+gem "decidim", DECIDIM_VERSION
 gem "decidim-conferences", DECIDIM_VERSION
-gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer.git", branch: "release/0.25-stable"
+gem "decidim-consultations", DECIDIM_VERSION
+gem "decidim-file_authorization_handler", git: "https://github.com/CodiTramuntana/decidim-file_authorization_handler.git", tag: "v0.26.2.3"
+gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer.git", branch: "release/0.26-stable"
 gem "decidim-verifications-members_picker", git: "https://github.com/gencat/decidim-verifications-members_picker.git", tag: "0.0.2"
 
-gem "daemons"
-gem "delayed_job_active_record"
```
Comment (tramuntanal) [inline Gemfile:13]: why remove delayed_jobs?

File: Gemfile [hunk 2]
```diff
@@ -4,26 +4,21 @@ source "https://rubygems.org"
...
-gem "puma"
-gem "uglifier", ">= 1.3.0"
-gem "whenever"
```
Comment (tramuntanal) [inline Gemfile:16]: why remove whenever?

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Just a couple of things to check please

**Review by tramuntanal — APPROVED** [review]
Good thanks!

**Pull request merged**